crates/uv/src/commands/tool/common.rs

```diff
@@ -255,83 +258,94 @@ pub(crate) fn install_executables(
     #[cfg(windows)]
     let itself = std::env::current_exe().ok();
 
-    for (name, source_path, target_path) in &target_entry_points {
+    let mut names = BTreeSet::new();
+    for (name, src, dst) in &target_entry_points {
         debug!("Installing executable: `{name}`");
 
         #[cfg(unix)]
-        replace_symlink(source_path, target_path).context("Failed to install executable")?;
+        replace_symlink(src, dst).context("Failed to install executable")?;
 
         #[cfg(windows)]
-        if itself.as_ref().is_some_and(|itself| {
-            std::path::absolute(target_path).is_ok_and(|target| *itself == target)
-        }) {
-            self_replace::self_replace(source_path).context("Failed to install entrypoint")?;
+        if itself
+            .as_ref()
+            .is_some_and(|itself| std::path::absolute(dst).is_ok_and(|target| *itself == target))
+        {
+            self_replace::self_replace(src).context("Failed to install entrypoint")?;
         } else {
-            fs_err::copy(source_path, target_path).context("Failed to install entrypoint")?;
+            fs_err::copy(src, dst).context("Failed to install entrypoint")?;
         }
+
+        names.insert(name.trim_end_matches(EXE_SUFFIX));
     }
 
-    let s = if target_entry_points.len() == 1 {
-        ""
+    let plural = if names.len() == 1 { "" } else { "s" };
```

Comment:
Nit: We use `s` as the name pretty consistently in this repository, it'd be nice to retain the style there.